api/routes.go

```diff
@@ -20,6 +20,9 @@ const (
 	usersEndpoint = "/users"
 	// POST /users/verify to verify the user
 	verifyUserEndpoint = "/users/verify"
+	// GET /users/verify/code to get the user verification code information
+	// POST /users/verify/code to try to resend the user verification code
+	verifyUserCodeEndpoint = "/users/verify/code"
```

Comment:
@lucasmenendez shouldn't this endpoint have a more obvious name? 
If I understand correctly the fucntion, something like `retry` or similar would make more sense

Reply:
It is also used to get the expiration time of an existing verification code, so here is the HTTP verb that adds more information about the endpoint (`GET` to retrieve info about the code and `POST` to create a new code)